src/cart/src/cart/crt_context.c

```diff
@@ -1020,14 +973,6 @@ crt_context_req_untrack(struct crt_rpc_priv *rpc_priv)
 		tmp_rpc = d_list_entry(epi->epi_req_waitq.next,
 					struct crt_rpc_priv, crp_epi_link);
 		tmp_rpc->crp_state = RPC_STATE_INITED;
-		crt_set_timeout(tmp_rpc);
```

Comment:
is this intentional change to start tracking timeout from the moment rpc was submitted by user vs rpc being on a wire?

Reply:
Hmm. Maybe I removed it by mistake... Will look at this. Thanks to point this!